Dockerfile

```diff
@@ -4,8 +4,8 @@ ARG FFMPEG_VERSION=latest
 FROM node:alpine as web-builder
 ARG JELLYFIN_WEB_VERSION=10.5.1
 RUN apk add curl git \
- && curl -L https://github.com/jellyfin/jellyfin-web/archive/${JELLYFIN_WEB_VERSION}.tar.gz | tar zxf - \
- && cd jellyfin-web-* \
+ && git clone --branch release-10.5.z --single-branch https://github.com/jellyfin/jellyfin-web.git \
```

Comment:
We can't hardcode this stuff in. It has to be fully dynamic, or it will break boht nightlies and future releases.